cli/src/merge_tools/external.rs

```diff
@@ -344,27 +344,41 @@ pub fn run_mergetool_external(
     Ok(tree_builder.write_tree())
 }
 
+// Not interested in $UPPER_CASE_VARIABLES
+static VARIABLE_REGEX: once_cell::sync::Lazy<Result<Regex, regex::Error>> =
+    once_cell::sync::Lazy::new(|| Regex::new(r"\$([a-z0-9_]+)\b"));
```

Comment:
nit: `.unwrap()` here since the const regexp must be parsable.

Reply:
I think you were looking at an old version, this is already fixed. Let me know if you'd like more changes.